### DISCUSSION TITLE
Audio setup item missing

### DISCUSSION THREAD

**Discussion by jwings333** [GrandOrgue/grandorgue#1444] [General]
I have been using GrandOrgue on centos 7.x for a while, so I am used to setting up multichannel audio via the menu item. It works quite well well with jack audio and a GigaExpress 8 channel box.

I am now trying to get GO running on Rocky 8.7. The .appimage and rpm work fine, but there seems to be NO menu item for setting the audio channels. I can't find anything on the net to explain where it has gone. BTW, with the std stereo output, all output comes out fine, so the instrument is working. I can also connect with the midi keyboards and pedal.

With the move to RHEL 8, the audio stream is all done through pipestream. I have been able via the KDE audio to push sample sounds to the various speakers used for GO output (three pairs + sub). 

I really appreciate having the instant setup (via rpm or appimage), but this issue has me stumped.

**Comment by oleg68** [comment]
The right place for distributing GO audio output between several channels is File->Settings->Audio

**Reply by jwings333** [reply]
Yes, I found the settings menu. In prior versions it was under audio/midi settings one top menu over. Even when hitting the settings item, it was slow to come up and was hidden behind the console pattern.

That being said, I ditched pipewire since it was too immature and not well supported in rhel 8. So I went back to direct use of jackd. Between the audio menu and the jackd connection, I now have six channel audio again! Now I am satisfied with the combo and will put it all on a stand alone Rocky 8 Linux box.
